lib/src/mustachio/renderer_base.dart

```diff
@@ -230,13 +230,20 @@ abstract class RendererBase<T> {
     }
 
     if (property.renderIterable != null) {
-      // An inverted section is rendered with the current context.
-      if (node.invert && property.isEmptyIterable(context)) {
+      var renderedIterable =
+          property.renderIterable(context, this, node.children);
+      if (node.invert && renderedIterable.isEmpty) {
+        // An inverted section is rendered with the current context.
         renderBlock(node.children);
+      } else if (!node.invert && renderedIterable.isNotEmpty) {
+        var buffer = StringBuffer();
+        for (var renderedElement in renderedIterable) {
+          buffer.write(renderedElement);
+        }
```

Comment:
Can this be simplified to the following?

```dart
buffer.writeAll(renderedIterable);
```

Reply:
I love it. Done.